templates/eos/template.json

```diff
@@ -19,13 +19,5 @@
             "default": "/${PROJECT_NAME}/\\W+/;/^;+//\\w+/\\L\\0\\E/[^;]$/\\0;/",
             "pattern": "^(\\w+;)+$"
         }
-    },
```

Comment:
See other comment. This change and all similar to it needs to be reverted.

Reply:
Fixed